.goreleaser/dockers.yaml

```diff
@@ -56,6 +56,7 @@ docker_manifests:
     skip_push: "{{ .Env.SKIP_PUSH_TEST_AGENT }}"
 
 docker_signs:
-  - cmd: ./bin/cosign
+  - cmd: go
+    args: ["tool", "cosign", "sign", "--key=cosign.key", "${artifact}@${digest}", "--yes"]
```

Comment:
Bit unfortunate, but we apparently can't template here: https://goreleaser.com/customization/docker_sign/

Reply:
This does seem pretty readable at least

Reply:
That's true